models/inventory/StockTransfer.ts

```diff
@@ -246,6 +272,47 @@ export abstract class StockTransfer extends Transfer {
     await invoice.setAndSync('stockNotTransferred', notTransferred);
   }
 
+  async _updateItemsReturned() {
+    if (this.isSyncing || !this.returnAgainst) {
```

Comment:
Why is there a check for `isSyncing`? If this is required then there is an `await` missing somewhere.

Reply:
seems like it's not required. I can push the fixes after a proper check if that's okay.